openreview/conference/matching.py

```diff
@@ -251,6 +251,91 @@ def _build_conflicts(self, submissions, user_profiles, get_profile_info):
             return self._build_profile_conflicts(ac_profiles, user_profiles)
         return self._build_note_conflicts(submissions, user_profiles, get_profile_info)
 
+    def append_note_conflicts(self, profile_id, build_conflicts=None):
+        '''
+        Create conflict edges between the given Notes and a single profile
+        '''
+
+        # Re-setup information that would have been initialized in setup()
+        submissions = list(openreview.tools.iterget_notes(
+            self.conference.client,
+            invitation=self.conference.get_blind_submission_id(),
+            details='original'))
+        get_profile_info = openreview.tools.get_neurips_profile_info if build_conflicts == 'neurips' else openreview.tools.get_profile_info
+        
+        # Adapt single profile to multi-profile code
+        user_profiles = [profile_id]
+        user_profiles = _get_profiles(self.client, user_profiles, with_publications=build_conflicts)
+        # Check for existing OpenReview profile - perform dummy check
+        if user_profiles[0].active == None:
```

Comment:
can you add this validation at the beginning of the function? so we don't nee to load all the submissions if the profile doesn't exists.